lib/evmone/execution_state.hpp

```diff
@@ -84,23 +90,43 @@ struct ExecutionState
     int64_t gas_left = 0;
     evm_stack stack;
     evm_memory memory;
-    const evmc_message& msg;
+    const evmc_message* msg = nullptr;
     evmc::HostContext host;
-    const evmc_revision rev = {};
+    evmc_revision rev = {};
     bytes return_data;
-    const bytes_view code;
+    bytes_view code;
     evmc_status_code status = EVMC_SUCCESS;
     size_t output_offset = 0;
     size_t output_size = 0;
 
+    ExecutionState() noexcept = default;
```

Comment:
Is it needed?

Reply:
I believe Silkwarm first creates a ExecutionState in a pool with `std::make_unique<ExecutionState>()`.

Reply:
> Is it needed?

Yes, the default constructor is used in [Silkworm](https://github.com/torquem-ch/silkworm/blob/master/silkworm/execution/state_pool.cpp).